### ISSUE TITLE
After update this morning the integration no longer works

### ISSUE THREAD

**Issue by Jellybabz** [sockless-coding/panasonic_cc#312]
### Checklist

- [X] I have verified that the account and device works in the Comfort Cloud App.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/sockless-coding/panasonic_cc/issues?q=is%3Aissue+label%3A"bug"+)..

### What version of Home Assistant Core?

core-2024.9.2

### Describe the issue

After the update (2024.9.1) all my devices won't go online. It just days "Failed to set up" 

### Error/Debug Logs

```text
Logger: homeassistant.config_entries
Source: config_entries.py:604
First occurred: 06:54:23 (1 occurrences)
Last logged: 06:54:23

Error setting up entry for panasonic_cc
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 604, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/panasonic_cc/__init__.py", line 82, in async_setup_entry
    {"\n ".join(obj.raw for obj in api.unknown_devices)}
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: sequence item 0: expected str instance, dict found
```

**Comment by sockless-coding** [comment]
@Jellybabz I will fix that, it's a bug I managed to add when adding the extra Aquarea logging. You can downgrade to 2024.9.0 since it doesn't contain that bug.

**Comment by Jellybabz** [comment]
So if i don't know how to downgrade i will just have to wait for an update to fix the bug? :D

**Comment by sockless-coding** [comment]
Should be fixed in 2024.9.2
